README3.adoc

```diff
@@ -1052,10 +1052,36 @@ class MyActivity : Activity(), LazyKodeinAware {
 
 === Android scopes
 
+==== The context scope
+
+There are times where you need an object to be a singleton, but only during the lifetime of a `Context`. You can use the `contextSingleton` scope to achieve this.
+
+The context scope should be used when a binding could apply to either an `Activity` or a `Service`. When a binding is exclusively for an `Activity` or a `Service`, the activity or service scope should be used instead.
+
+[source, kotlin]
+.Example: using the context scope
+----
+val kodein = Kodein {
+    bind<Logger>() with contextSingleton { LogManager.getNamedLogger(it.localClassName) } // <1>
+}
+----
+<1> `it` is the context the object is being created for.
+
+To retrieve an object bound in the context scope, you need to inject a factory that takes the context as a parameter.
+
+[source, kotlin]
+.Example: retrieving a context scoped singleton
+----
+val logger = kodein.with(context).instance()
+val sameLogger = kodein.with(context).instance() // this will be the same object as logger
+val otherLogger = kodein.with(otherContext).instance() // this will be a different object than logger
+----
+
+NOTE: The activity and service scope are special cases of the context scope. The `ScopeRegistry` returned for an `Activity` or `Service` object from the context scope will be the same one returned for that object from the activity or service scope
```

Comment:
The `ScopeRegistry` is covered in the "advanced" section of this document. At this point, the reader does not know what it is. Therefore, the document should not mention it at this place.

The note is important, though. It just need to be rephrased without `ScopeRegistry` ;)